src/AutoMerge/AutoMerge.jl

```diff
@@ -10,6 +10,8 @@ import JSON
 
 import ..RegistryCI
 
+const AUTOMERGE_FAILURE_LABEL = "automerge failed"
```

Comment:
I considered "needs manual merge" here, but I think most of the time the automerge fails, the user will want to update the PR to obey the guidelines. One problem with "automerge failed" though is it sounds like something went wrong in the process (like an exception) instead of an intended possible outcome. Maybe "guidelines not met" or something?

Reply:
I think "automerge failed" is a fine name.